tiledb/sm/rest/curl.cc

```diff
@@ -302,28 +302,22 @@ Status Curl::init(
     return LOG_STATUS(Status_RestError(
         "Error initializing libcurl; failed to set CURLOPT_HEADERDATA"));
 
-  // Ignore ssl validation if the user has set rest.ignore_ssl_validation = true
-  bool ignore_ssl_validation = false;
-  bool found;
-  RETURN_NOT_OK(config_->get<bool>(
-      "rest.ignore_ssl_validation", &ignore_ssl_validation, &found));
```

Comment:
Same as in the comment above, I understand we now support both for backwards compatibility, but do we want to keep this one around indefinitely in the future? If no, should we discourage users? can we clarify somehow that this was enhanced and replaced by another option so it doesn't get very confusion which option gets the job done?